packages/esm-patient-chart-app/src/workspace/workspace-window.component.tsx

```diff
@@ -13,7 +13,8 @@ interface ContextWorkspaceParams {
   patientUuid?: string;
 }
 
-const WorkspaceWindow: React.FC<ContextWorkspaceParams> = ({ patientUuid }) => {
+const WorkspaceWindow: React.FC<ContextWorkspaceParams> = () => {
```

Comment:
Changed from a parameter to using a hook. `usePatient()` itself should basically guarantee that we're only fetching as appropriate

Reply:
Empirically, not really for me at least relative to the delay it had before hand, but yeah, I was hoping `usePatient()` would deduplicate things, but it doesn't.